src/components/themed/FilledTextInput.tsx

```diff
@@ -208,9 +209,9 @@ export const FilledTextInput = React.forwardRef<FilledTextInputRef, FilledTextIn
   const InputComponent = numeric ? StyledNumericInput : StyledAnimatedTextInput
 
   return (
-    <>
+    <View style={spaceStyle}>
```

Comment:
Why was this done? Isn't the whole point of using the space props within the component to avoid the extra view? If this is our result, we might as well just use a <Space> component around the input when we want the space props, at least then we'd only be rendering an extra view when we actually want the space.

Reply:
This fixes the bug where extra margin was added by a space prop but inbetween the input box and the bottom right message. it's too late in the sprint to make a change that would break the API given how many places use this component

Reply:
In case, we can just use a Space component and pass the space props directly to it. Also, override sideways to false.